### PR TITLE
Add arguments to create_db

### LINKED ISSUE #4704: Create_db always prompts for input
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-server

**Current behavior:**

Currently at create_db.py, it prompts in terminal for an e-mail address and a password, that all is fine, but currently there is no way to pass down variables from the terminal or from the enviroment so that it can decide beforehand which values to take from e-mail and password
**Expected behavior:**

I should be able to create databases from a script using predetermined values.
**Related code:**

```
insert any relevant code here else remove this section
```
https://github.com/fossasia/open-event-server/blob/e0a63f37e0ce7c7a39718f005a959f4194e6ada4/create_db.py#L30

### PR DESCRIPTION
**Pull request by Kreijstal**
Fixes #4704

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Basically you can optionally type the email and password arguments in the command line without needed to be propmpted for it

#### Changes proposed in this pull request:

- Add a condition that checks for wether the parameters have been called, if not, continue the same behaviour as before.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@Kreijstal Please `Allow maintainers to push commits` from next time so that users can collaborate on PRs and fix quick stylistic issues as pointed by the Hound CI
